src/apify_client/clients/resource_clients/task.py

```diff
@@ -0,0 +1,214 @@
+        memory_mb: Optional[int] = None,
+        timeout_secs: Optional[int] = None,
+    ) -> Dict:
+        """Update the task with specified fields.
+
+        https://docs.apify.com/api/v2#/reference/actor-tasks/task-object/update-task
+
+        Args:
+            name (string, optional): Name of the task
+            build (str, optional): Specifies the actor build to run. It can be either a build tag or build number.
+                                   By default, the run uses the build specified in the task settings (typically latest).
+            memory_mb (int, optional): Memory limit for the run, in megabytes. By default, the run uses a memory limit specified in the task settings.
+            timeout_secs: (int, optional): Optional timeout for the run, in seconds. By default, the run uses timeout specified in the task settings.
+            task_input (dict, optional): Task input dictionary
+
+        Returns:
+            The updated task
+        """
+        new_fields = {
+            "name": name,
+            "options": {
+                "build": build,
+                "memoryMbytes": memory_mb,
+                "timeoutSecs": timeout_secs,
+            },
+            "input": task_input,
+        }
+
+        return self._update(_filter_out_none_values_recursively(new_fields))
+
+    def delete(self) -> None:
+        """Delete the task.
+
+        https://docs.apify.com/api/v2#/reference/actor-tasks/task-object/delete-task
+        """
+        return self._delete()
+
+    def start(
+        self,
+        *,
+        task_input: Optional[Dict[str, Any]] = None,
+        build: Optional[str] = None,
+        memory_mb: Optional[int] = None,
+        timeout_secs: Optional[int] = None,
+        wait_for_finish: Optional[int] = None,
+        webhooks: Optional[List[Dict]] = None,
+    ) -> Dict:
+        """Start a task and immediately return the Run object.
+
+        https://docs.apify.com/api/v2#/reference/actor-tasks/run-collection/run-task
+
+        Args:
+            task_input (dict, optional): Task input dictionary
+            build (str, optional): Specifies the actor build to run. It can be either a build tag or build number.
+                                   By default, the run uses the build specified in the task settings (typically latest).
+            memory_mb (int, optional): Memory limit for the run, in megabytes. By default, the run uses a memory limit specified in the task settings.
+            timeout_secs: (int, optional): Optional timeout for the run, in seconds. By default, the run uses timeout specified in the task settings.
+            wait_for_finish: (bool, optional): The maximum number of seconds the server waits for the run to finish.
+                                               By default, it is 0, the maximum value is 300.
+            webhooks (list, optional): Specifies optional webhooks associated with the actor run, which can be used to receive a notification
```

Comment:
The same issue as with the "build" parameter comments, also could do without the "Note:", since the entire comment is a note ;)

Reply:
Also, it wouldn't be clear what format the webhooks should be in if I were a user.